src/lib/courses.ts

```diff
@@ -124,7 +124,7 @@ export async function loadCourseMetadata(id: number, slug: string) {
 
   const course = await sanityClient.fetch(courseQuery, params)
 
-  if (!course?.square_cover_480_url) {
+  if (course && !course?.square_cover_480_url) {
```

Comment:
in the rare case a course is not found in sanity, this ensures we aren't attempting to modify a null value